SharedSettings/Catalyst.ClassLibrary.nuspec

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>Catalyst.MinimumRecommendedRules</id>
+    <title>Catalyst shared Code Analisys Rules: Class Library rules</title>
+    <version>$version$</version>
+    <authors>Health Catalyst</authors>
+    <owners>Health Catalyst</owners>
+    <licenseUrl>https://github.com/HealthCatalyst/Fabric.SharedSettings</licenseUrl>
+    <projectUrl>https://github.com/HealthCatalyst/Fabric.SharedSettings</projectUrl>
+    <description>Deployment package for shared code analysis rules. Health Catalyst rules that make sense in class library projects.</description>
+  </metadata>
```

Comment:
Let's use developmentDependency=true (along with the appropriate minClientVersion) so this NuGet package doesn't get included as a downstream NuGet dependency. 
https://docs.microsoft.com/en-us/nuget/reference/nuspec

Reply:
This has been updated. It's ready for re-review now @j2jensen